test/quickstart.sh

```diff
@@ -65,7 +65,7 @@ curl -s -X GET "http://localhost:8000/openai/v1/models" | jq '. | length == 0'
 # By using the --reuse-values flag we can just append models to the previous install
 # while avoiding overriding the image that skaffold originally built and set in the
 # first install.
-helm upgrade --reuse-values --install kubeai charts/kubeai -f - <<EOF
+helm upgrade --reuse-values --install kubeai ./charts/kubeai -f - <<EOF
```

Comment:
Curious was this needed due to being on Mac?

Reply:
No, this didnt solve anything - but it made it more clear for me that this was referencing a local dir